Signal/src/Signal-Bridging-Header.h

```diff
@@ -2,13 +2,16 @@
 //  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
 //
 
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+
+// Separate iOS Frameworks from other imports.
```

Comment:
Adding a comment separator ensures that we preserve that proper ordering of header imports.